doc/architecture/database_schemata.md

```diff
@@ -149,3 +203,58 @@ subfamily }|--|| family : ""
+
+## `Maps`
+
+| **_Column name_**                | **_Example_**                    | **_Initial rule_**                                                                                | **_Description_**                  |
+| :------------------------------- | :------------------------------- | :------------------------------------------------------------------------------------------------ | :--------------------------------- |
+| **id**                           | 1                                |
+| **owner_id**                     | 1                                |
+| **name**                         | My Map                           |
+| **is_variant**                   | false                            |
+| **version_name**                 | NULL                             |
+| **is_inactive**                  | false                            |
+| **last_visit**                   | 2023-04-04                       |
+| **honors**                       | 0                                | | 0 to infinity
+| **visits**                       | 0                                | | 0 to infinity
+| **harvested**                    | 0                                | | 0 to infinity
+| **creation_date**                | 2023-04-04                       |
+| **deletion_date**                | 2023-04-04                       |
+| **zoom_factor**                  | 100                              | | value used in formula "X by X cm", e.g. 100 would mean "100 x 100 cm", range from 10 to 100000
+| **background**                   | NULL                             |
+| **background_scale**             | NULL                             |
+| **geo_data**                     | NULL                             | | PostGis Geodata
+| **epsg_code**                    | NULL                             | | PostGis EPSG Code
+
+## `Users`
+
+| **_Column name_**                | **_Example_**                    | **_Initial rule_**                                                                                | **_Description_**                  |
+| :------------------------------- | :------------------------------- | :------------------------------------------------------------------------------------------------ | :--------------------------------- |
+| **id**                           | 1                                |
+| **nc_uid**                       | 1                                | | Nextcloud ID
+| **free_until**                   | 2023-04-04                       |
+| **app_language**                 | English                          |
+| **member_since**                 | 2023-04-04                       |
+| **experience**                   | {beginner, hobbyist, expert}     |
+| **preferences**                  | raised vegetable beds            |
+| **location**                     | Vienna, Austria                  |
+| **membership_points**            | 0                                | | 0 to infinity
+
+## `Blossoms`
```

Comment:
Should we add "blossoms" to the glossary?

Reply:
I will do that today. Thanks for reminding me, I meant to do that a while ago already.